### DISCUSSION TITLE
Add multiple domain feature.

### DISCUSSION THREAD

**Discussion by wahyulingu** [laravel/valet#1327] [Ideas]
I thought instead of using a single domain like .test or .local how about adding a feature to support multiple domains at once?

that way this tool can be used to manage multiple laravel/nginx deployments on one vps server.

I'm using google translate

**Comment by mattstauffer** [comment]
I think you mean multiple TLDs; that's not something I intend to support. Valet is not for production use.

**Comment by ZebTheWizard** [comment]
I made a fork which modifies valet to add some quick and dirty support for multiple TLDs.  It's not great but is useful when working with strict oauth2 workflows.  Think there are some limitations with how valet recognizes sites.  Currently it symbolically links based on the name of the site without the TLD so there is not an elegant way to determine what is a site and what is a proxy.  Valet could probably benefit from an sqlite db.

https://github.com/SnowLaboratory/valet
